docs/metrics/object-detection.md

```diff
@@ -0,0 +1,206 @@
+   ""label"": ""truck"", ""data_type"": ""ANNOTATION/BOUNDING_BOX""}]"
+```
+The expected CSV format for this dataset should look like the CSV below:
+```
+locator,image_id,ground_truths,extra_information_as_metadata
+s3://my-images/intersection_1.png,1,"[{""top_left"": [538.03, 8.86], ... }, {""top_left"": ...}]",cloudy
+s3://my-images/intersection_2.png,2,"[{""top_left"": [1380.35, 4.84], ... }]",rainy
+...
+```
+This CSV can be directly uploaded to Kolena's platform. To upload a dataset with the SDK, see details for
+[structured data usage](../dataset/formatting-your-datasets.md/#structured-data).
+
+Once your dataset and ground truths are uploaded, you can upload model inferences and results in a similar fashion.
+An object detection model will predict the bounding boxes of objects within each image, so there is a list of
+inferences per image:
+[`List[ScoredLabeledBoundingBox]`](../reference/annotation.md#kolena.annotation.ScoredLabeledBoundingBox).
+
+<div class="grid" markdown>
+<figure markdown>
+  ![OD_metrics.png](../assets/images/metrics-tasks-od-qs-light.png#only-light)
+  ![OD_metrics.png](../assets/images/metrics-tasks-od-qs-dark.png#only-dark)
+  <figcaption markdown>Configurable metrics for custom [Quality Standards](../dataset/core-concepts/quality-standard.md)
+  </figcaption>
+</figure>
+
+!!! info "Built-In Object Detection Metrics"
+
+    Rather than only uploading raw inferences, it is also worthwhile to upload metrics such as
+    [TP / FP / FN](./tp-fp-fn-tn.md) from your own [bounding box matcher](./geometry-matching.md) as counts and as
+    lists of annotations so that they appear in their respective groups within the Studio.
+
+    Furthermore, Kolena can automatically compute common object detection metrics such as
+    [precision](./precision.md) and [recall](./recall.md), making it easy to visualize an assortment of plots
+    on the fly.
+
+    ![OD_metrics.png](../assets/images/metrics-tasks-od-debugger-light.png#only-light)
+    ![OD_metrics.png](../assets/images/metrics-tasks-od-debugger-dark.png#only-dark)
+
+    In the debugger, you are able to see plots by class or by test case as shown above. For more details on
+    automatic metrics and plots, please refer to documentation for
+    [formatting results](../dataset/formatting-your-datasets.md/#formatting-results-for-object-detection).
+
+</div>
+
+Note that a [`ScoredLabeledBoundingBox`](../reference/annotation.md#kolena.annotation.ScoredLabeledBoundingBox)
+has an extra `score` attribute for the confidence of the model:
+```
+{""top_left"": ... ""label"": ""car"", ""score"": 0.94, ""data_type"":""ANNOTATION/BOUNDING_BOX""}
+```
+
+The expected CSV format for model results should look like the CSV below:
+```
+image_id,count_TP,count_FP,count_FN,TP,FP,FN,custom_information
+1,2,0,0,"[{""top_left"": [538.5, 9.0], ... }, {""top_left"": ... }]","[]","[]",0.88
+2,0,0,1,"[]","[]","[{""top_left"": ... }]",0.5
+...
+```
```

Comment:
It would be nice to add a paragraph about what needs to be configured on Debugger / QS metrics, e.g. what columns do I pick for these?:
<img width="328" alt="image" src="https://github.com/kolenaIO/kolena/assets/17771952/21c25f98-ca59-4170-8b91-c3c1f42a53b4">